docs/source/sysadmin_guide/configuring_permissions.rst

```diff
@@ -84,106 +121,114 @@ transactions. Each role equals a policy name that corresponds to a policy
-  :language: console
+     $ sawset proposal create sawtooth.identity.allowed_keys=02b2be336a6ada8f96881cd55fd848c10386d99d0a05e1778d2fc1c60c2783c2f4
 
-.. literalinclude:: ../cli/output/sawtooth_identity_role_create_usage.out
-  :language: console
+#. Once your signer key is stored in the setting, use the command
+   ``sawtooth identity policy create`` to set and update roles and policies.
 
-For example, running the following will create a policy that permits all
-and is named policy_1:
+   For example, running the following command will create a policy that permits all
+   and is named ``policy_1``:
 
-.. code-block:: console
+   .. code-block:: console
 
-    $ sawtooth identity policy create policy_1 "PERMIT_KEY *"
+       $ sawtooth identity policy create policy_1 "PERMIT_KEY *"
 
-To see the policy in state, run the following command:
+   You can enter multiple permit or deny rules in the same command; separate
+   each rule with a space.
 
-.. code-block:: console
+   .. important::
 
-  $ sawtooth identity policy list
-  policy_1:
-    Entries:
-      PERMIT_KEY *
+      Be careful not to remove your permission to change Identity settings.
+      Sawtooth will not prevent you from entering a rule to deny all transactors
+      (including yourself).
 
-Finally, run the following command to set the role for transactor to permit all:
+#. To see the policy in state, run the following command:
 
-.. code-block:: console
+   .. code-block:: console
 
-    $ sawtooth identity role create transactor policy_1
+     $ sawtooth identity policy list
+     policy_1:
+       Entries:
+         PERMIT_KEY *
 
-To see the role in state, run the following command:
+#. Use the command ``sawtooth identity role create`` to create a role
+   for this policy. The following example sets the role for ``transactor``
+   to the policy that permits all:
 
-.. code-block:: console
+   .. code-block:: console
 
-    $ sawtooth identity role list
-    transactor: policy_1
+       $ sawtooth identity role create transactor policy_1
+
+#. To see the role in state, run the following command:
+
+   .. code-block:: console
+
+       $ sawtooth identity role list
+       transactor: policy_1
+
+.. _transactor-roles-label:
 
 Transactor Roles
 ----------------
-The following are the identity roles that are used to control which transactors
+The following identity roles are used to control which transactors
 are allowed to sign transactions and batches on the system.
 
-default:
+``default``:
   When evaluating role permissions, if the role has not been set, the default
-  policy will be used. The policy can be changed to meet the network's
+  policy is used. The policy can be changed to meet the network's
   requirements after initial start-up by submitting a new policy with the name
   default. If the default policy has not been explicitly set, the default
-  is “PERMIT_KEY \*”.
+  is ``PERMIT_KEY \*``.
```

Comment:
I think "\" needs to be remvoed

Reply:
I shall remove the now-unnecessary backslash.